src/utils/transform.test.ts

```diff
@@ -110,7 +286,7 @@ const result = new MutableDataFrame({
         '1-5ee20a4a-bab71b6bbc0660dba2adab3e',
         '1-5ee20a4a-bab71b6bbc0660dba2adab3e',
         '1-5ee20a4a-bab71b6bbc0660dba2adab3e',
-        undefined,
+        '',
```

Comment:
> Trace ID of the subsegment's parent segment. Required only if sending a subsegment separately.
> [Subsegment fields](https://docs.aws.amazon.com/xray/latest/devguide/xray-api-segmentdocuments.html#api-segmentdocuments-subsegments)

From my understanding of the docs, and from the payload we receive from X-Ray, this can be empty because the trace id is only sent if a subsegment is sent separately, otherwise this field is empty. By sent separately, it's something like this

```json
{
  ID: 'trace-id-1',
  Segments: [
    {
      Document: {
        id: 'segment-id-1',
        trace_id: 'trace-id-1',
        subsegments: [
          // a subsegment of 'segment-id-1'.
          {
            id: 'subsegment-id-1'
            // trace_id is not a part of this subsegment because it doesn't have any subsegments that are sent separately
          }
        ],
      },
     {
       // this is a subsegment of segment-id-1 that is sent separately
       Document: {
         id: 'subsegment-id-1',
         trace_id: 'trace-id-1',
       },
}
```